sunpy/net/jsoc/jsoc.py

```diff
@@ -40,8 +40,8 @@ def __init__(self, table=None, client=None):
         """
         super().__init__()
         self.table = table or astropy.table.QTable()
```

Comment:
The `if isinstance(table, list) and isinstance(table[0], type(self)):` is not here now but it didn't seem to break anything so ?

Reply:
Fun fact, this was needed on the backports but not on master with the Fido changes.